src/modules/directory.rs

```diff
@@ -129,6 +109,38 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
+fn is_truncated(path: &str) -> bool {
+    !(path.starts_with(HOME_SYMBOL)
+        || if cfg!(not(target_os = "windows")) {
+            path.starts_with('/')
+        } else {
+            PathBuf::from(path).has_root() || PathBuf::from(String::from(path) + r"\").has_root()
```

Comment:
Any reason `has_root` is only used on Windows?

Reply:
At this point it was a string, so rather than creating an object just to find if it's root, I just looked for the leading slash.

The Windows version has to use the `has_root` because a root path on windows could start with any drive letter, or be a network path.

It's an easy change though, so I will make it and test it.